src/WatchDogs.Contracts/Contracts.csproj

```diff
@@ -6,4 +6,8 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Serilog.AspNetCore" Version="8.0.1" />
```

Comment:
Please remove this, since our Contracts should not depend on implementation details like log provider

Reply:
Done.